saveSvgAsPng.js

```diff
@@ -127,6 +127,14 @@
 
               if (externalFontUrl) {
                 // okay, we are lucky. We can fetch this font later
+
+                //handle url if relative
+                if (externalFontUrl.startsWith('../')) {
+                  externalFontUrl = sheets[i].href + '/../' + externalFontUrl
+                } else if (externalFontUrl.startsWith('./')) {
+                  externalFontUrl = sheets[i].href + '/.' + externalFontUrl
```

Comment:
I'm not sure this is right. It looks like it turns a relative URL with `../` into an absolute URL with `/../../`. Is that what it's supposed to do?

Also, turns `./` into `/../`?